pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -384,7 +384,7 @@ type WorkloadObjectMeta struct {
 // +genclient
 // +kubebuilder:object:root=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +kubebuilder:resource:singular="podinfo",path="podinfo",scope="Namespaced",shortName={}
+// +kubebuilder:resource:categories={tetragon},singular="podinfo",path="podinfo",scope="Namespaced",shortName={tgpi}
```

Comment:
I'm thinking it might make sense to omit PodInfo CRD from this category, as it's an "internal" CRDs (i.e. users shouldn't need to care about PodInfo at all). But it's fine to keep it for now.

Reply:
I just removed it in the latest force push. I think this PR is ready to be merged as soon as all tests are completed.